src/renderers/shared/fiber/ReactReifiedYield.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactReifiedYield
+ * @flow
+ */
+
+'use strict';
+
+import type { ReactYield } from 'ReactCoroutine';
+import type { Fiber } from 'ReactFiber';
+
+var ReactFiber = require('ReactFiber');
+
+type ReifiedYield = { continuation: Object, props: Object };
```

Comment:
This won't have the security implications and it won't be used as a terminal that we have to test for.

I guess the key is that this never returns back into React. I think of it as the key/value pair exposed by `Map`. I think we should probably just make this an array tuple to optimize property access too.